app/services/PsrRetrievalService.scala

```diff
@@ -38,11 +39,13 @@ class PsrRetrievalService @Inject()(
   declarationTransformer: DeclarationTransformer
 ) {
 
-  def getStandardPsrDetails(
-    optFbNumber: Option[String],
-    optPeriodStartDate: Option[String],
-    optPsrVersion: Option[String],
-    fallBackCall: Call
+  // todo should return Option.None on 404 instead of empty user answers
```

Comment:
Is this todo to be completed in this ticket, or in a future ticket?

Reply:
Just an observation, will try and pick it up in a future ticket but the logic touches a few places so need to be careful